contents/blog/feedback-loops.md

```diff
@@ -0,0 +1,67 @@
+---
+date: 2021-12-15
+title: Aggressively seeking feedback loops for product development
```

Comment:
Potential titles: (did a quick SEO search, and the only close-ish reasonable term I found was "feedback loops software development" with meh volume)

How seeking feedback loops help build a better product

Software development using feedback loops

----

And more fun titles I like:

Reply:
Trust me, the braces are magic - it's a useful way to add emphasis and makes it feel more personal and fun.

I'm just wondering if there's a way to tweak to get it closer to the original sentiment...

- How I supercharged feedback loops to make a better product
- What's the secret to make better products? Aggressively seeking feedback loops
- How aggressively seeking feedback loops made me a better engineer

They're all ok, but still (2) best.

@joethreepwood any thoughts on the above?

Reply:
> Trust me, the braces are magic

Trusting you!